From 111bbd770441ab34f9da5bb1d85767a9b75227b4 Mon Sep 17 00:00:00 2001
From: Florian Fainelli <florian@openwrt.org>
Date: Mon, 28 Jan 2013 20:06:30 +0100
Subject: [PATCH 12/12] MIPS: BCM63XX: EHCI controller does not support
 overcurrent

This patch sets the ignore_oc flag for the BCM63XX EHCI controller as it
does not support proper overcurrent reporting.

Signed-off-by: Florian Fainelli <florian@openwrt.org>
---
 arch/mips/bcm63xx/dev-usb-ehci.c |    1 +
 1 file changed, 1 insertion(+)

--- a/arch/mips/bcm63xx/dev-usb-ehci.c
+++ b/arch/mips/bcm63xx/dev-usb-ehci.c
@@ -61,6 +61,7 @@ static void bcm63xx_ehci_power_off(struc
 static struct usb_ehci_pdata bcm63xx_ehci_pdata = {
 	.big_endian_desc	= 1,
 	.big_endian_mmio	= 1,
+	.ignore_oc		= 1,
 	.power_on		= bcm63xx_ehci_power_on,
 	.power_off		= bcm63xx_ehci_power_off,
 	.power_suspend		= bcm63xx_ehci_power_off,
